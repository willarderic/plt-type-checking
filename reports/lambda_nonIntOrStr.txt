TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: LAMBDA  LEXEME: lambda					TOKEN: LAMBDA    LEXEME: lambda

___Entering new scope... 					___Entering new scope...

TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
ID_LIST -> epsilon						ID_LIST -> epsilon
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: LAMBDA  LEXEME: lambda					TOKEN: LAMBDA    LEXEME: lambda

___Entering new scope... 					___Entering new scope...

TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
ID_LIST -> epsilon						ID_LIST -> epsilon
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
TOKEN: INTCONST  LEXEME: 42					TOKEN: INTCONST  LEXEME: 42
CONST -> INTCONST						CONST -> INTCONST
EXPR -> CONST							EXPR -> CONST
LAMBDA_EXPR -> lambda ( ID_LIST ) EXPR				LAMBDA_EXPR -> lambda ( ID_LIST ) EXPR

___Exiting scope...						___Exiting scope...

PARENTHESIZED_EXPR -> LAMBDA_EXPR				PARENTHESIZED_EXPR -> LAMBDA_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
LAMBDA_EXPR -> lambda ( ID_LIST ) EXPR				LAMBDA_EXPR -> lambda ( ID_LIST ) EXPR

___Exiting scope...						___Exiting scope...

Line 1: Arg 2 cannot be function				Line 1: Arg 2 cannot be function
