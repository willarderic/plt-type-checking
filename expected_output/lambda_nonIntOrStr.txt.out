TOKEN: LPAREN    LEXEME: (
TOKEN: LAMBDA    LEXEME: lambda

___Entering new scope...

TOKEN: LPAREN    LEXEME: (
ID_LIST -> epsilon
TOKEN: RPAREN    LEXEME: )
TOKEN: LPAREN    LEXEME: (
TOKEN: LAMBDA    LEXEME: lambda

___Entering new scope...

TOKEN: LPAREN    LEXEME: (
ID_LIST -> epsilon
TOKEN: RPAREN    LEXEME: )
TOKEN: INTCONST  LEXEME: 42
CONST -> INTCONST
EXPR -> CONST
LAMBDA_EXPR -> lambda ( ID_LIST ) EXPR

___Exiting scope...

PARENTHESIZED_EXPR -> LAMBDA_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
LAMBDA_EXPR -> lambda ( ID_LIST ) EXPR

___Exiting scope...

Line 1: Arg 2 cannot be function
